### ISSUE TITLE
kube-dns selinux AVCs

### ISSUE THREAD

**Issue by sjenning** [kubernetes-incubator/cri-o#911]
When running `hack/local-up-cluster.sh` during kubernetes development, kube-dns is deployed by default but fails to run due to blocked access on `/etc/resolv.conf`.  Also ran `ping` which needs ioctl access on that file (why?).  On F26:

```
AVC avc:  denied  { read } for  pid=11278 comm="kube-dns" name="resolv.conf" dev="tmpfs" ino=229066 scontext=system_u:system_r:container_t:s0:c422,c590 tcontext=system_u:object_r:container_var_run_t:s0 tclass=file permissive=0
AVC avc:  denied  { open } for  pid=15813 comm="kube-dns" path="/etc/resolv.conf" dev="tmpfs" ino=254881 scontext=system_u:system_r:container_t:s0:c160,c359 tcontext=system_u:object_r:container_var_run_t:s0 tclass=file permissive=0
AVC avc:  denied  { ioctl } for  pid=21722 comm="ping" path="/etc/resolv.conf" dev="tmpfs" ino=283126 ioctlcmd=0x5401 scontext=system_u:system_r:container_t:s0:c225,c880 tcontext=system_u:object_r:container_var_run_t:s0 tclass=file permissive=0
```

The following TE policy addresses this

```
module kubedns 1.0;

require {
        type container_var_run_t;
        type container_t;
        class file { open read ioctl };
}

#============= container_t ==============
allow container_t container_var_run_t:file read;
allow container_t container_var_run_t:file open;
allow container_t container_var_run_t:file ioctl;
```
@mrunalp @rhatdan @derekwaynecarr

**Comment by rhatdan** [comment]
https://github.com/kubernetes-incubator/cri-o/pull/903 Should fix this.  CRI-O needs to set the SELinux label correctly on resolv.conf and hostname.

**Comment by runcom** [comment]
Added fix this in #903, Seth could you test it out?
